ng2-bootstrap.ts

```diff
@@ -16,3 +31,24 @@ export * from  './components/typeahead';
 export * from  './components/position'
 export * from  './components/common'
 export * from  './components/ng2-bootstrap-config';
+
+export default {
+  directives: [Dropdown, Progress, Tab, TabHeading],
+  components: [
+    Accordion,
+    AccordionPanel,
```

Comment:
It injects components into directives, but you can add this to directives,
it will work the same.

On Thursday, 4 February 2016, Nathan Walker <notifications@github.com>
wrote:

> In ng2-bootstrap.ts
> <https://github.com/valor-software/ng2-bootstrap/pull/137#discussion_r51913825>
> :
>
> > @@ -16,3 +31,24 @@ export * from  './components/typeahead';
> >  export * from  './components/position'
> >  export * from  './components/common'
> >  export * from  './components/ng2-bootstrap-config';
> > +
> > +export default {
> > +  directives: [Dropdown, Progress, Tab, TabHeading],
> > +  components: [
> > +    Accordion,
> > +    AccordionPanel,
>
> So I believe all the components listed in this export should be in the
> directives collection altogether since they would in fact be part of the
> directives metadata in a user's source.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/valor-software/ng2-bootstrap/pull/137/files#r51913825>
> .
>